poe-tasks/poetry-connector-tasks.toml

```diff
@@ -135,6 +135,55 @@ help = "Get the language of the connector from its metadata.yaml file. Use with
 cmd = """yq eval '.data.connectorBuildOptions.baseImage' $PWD/metadata.yaml"""
 help = "Get the base image of the connector from its metadata.yaml file."
 
+[tool.poe.tasks.get-local-version]
+cmd = """yq eval '.data.dockerImageTag' $PWD/metadata.yaml"""
+help = "Get the version of the connector from its metadata.yaml file. Use with -qq to get just the version."
+
+[tool.poe.tasks.create-connector-registry-artifacts]
+shell = '''
+set -eu
+
+# Define required and optional files
+REQUIRED_FILES=(
+  "metadata.yaml"
+  "icon.svg"
+)
+
+OPTIONAL_FILES=(
+  "pyproject.toml"
+  "poetry.lock"
+  "poetry.toml"
+  "setup.py"
+  "requirements.txt"
+  "README.md"
+)
+
+VERSION=$(poe -qq get-local-version)
+CONNECTOR_NAME=$(poe -qq get-connector-name)
+
+# Create build directory
+mkdir -p "build/releases/${VERSION}"
+
+# Clear existing contents
+rm -rf "build/releases/${VERSION}"/*
+
+# Copy required files
+for file in "${REQUIRED_FILES[@]}"; do
+  cp "$file" "build/releases/${VERSION}/"
+done
+
+# Copy optional files if they exist
+for file in "${OPTIONAL_FILES[@]}"; do
+  if [ -f "$file" ]; then
+    cp "$file" "build/releases/${VERSION}/"
+  fi
+done
+
+# Generate registry JSON files
+${POE_GIT_DIR}/poe-tasks/create-connector-registry-json.sh "build/releases/${VERSION}"
+'''
+help = "Create local registry artifacts for this connector version in build/releases/{version}/"
```

Comment:
Move the help text declarations above the script declarations ("shell") so they are easier to read in context.